bundle/config/resources/secret_scope.go

```diff
@@ -0,0 +1,86 @@
+package resources
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/marshal"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+)
+
+type SecretScopePermissionLevel string
+
+// SecretScopePermission holds the permission level setting for a single principal.
+// Multiple of these can be defined on any secret scope.
+// Secret scopes permissions are mapped to Secret ACLs
+type SecretScopePermission struct {
+	Level SecretScopePermissionLevel `json:"level"`
+
+	UserName             string `json:"user_name,omitempty"`
+	ServicePrincipalName string `json:"service_principal_name,omitempty"`
+	GroupName            string `json:"group_name,omitempty"`
+}
+
+type SecretScope struct {
```

Comment:
Please include a comment describing why the fields are inlined vs embedding an SDK struct.